js/layout-javascript/news-feed-code.js

```diff
@@ -1669,6 +1730,13 @@ DynamicList.prototype.searchData = function(options) {
       $('#news-feed-list-wrapper-' + _this.data.id).html('');
 
       _this.modifiedListItems = _this.addSummaryData(searchedData);
+      _this.modifiedListItems = _this.Utils.Records.sortByField(
```

Comment:
Move the `{` to the previous line like so.

```suggestion
      _this.modifiedListItems = _this.Utils.Records.sortByField({
```